contracts/smart-account/interfaces/IAccountRecoveryModule.sol

```diff
@@ -271,7 +271,7 @@ interface IAccountRecoveryModule {
      * Should be called by the Smart Account
      * @param newSecurityDelay new security delay
      */
-    function setSecurityDelay(uint48 newSecurityDelay) external;
+    function setSecurityDelay(uint24 newSecurityDelay) external;
```

Comment:
The reason for modification here was to avoid potential overflow when the `securityDelay` is added to the `validAfter` param, and the `validAfter` is `uint48` as per 4337